libflux/src/core/ast/tests.rs

```diff
@@ -395,6 +395,18 @@ fn test_json_expression_statement() {
     let deserialized: Statement = serde_json::from_str(serialized.as_str()).unwrap();
     assert_eq!(deserialized, n)
 }
+
+#[test]
+fn test_json_tvartype() {
+    let n = MonoType::Tvar(TvarType {
+        base: BaseNode::default(),
+    });
+    let serialized = serde_json::to_string(&n).unwrap();
+    assert_eq!(serialized, r#"{"type":"TvarType"}"#);
```

Comment:
I must have missed this in review, but TvarType should have an identifier just like NamedType. Are you able to update it? The json should look like:

    {"type":"TvarType", "name":{"name": ...}}

Reply:
The test cases should also change in that case right?

Reply:
Yes.

Reply:
It looks like there might only be one test case that needs to be updated.